docs/tutorials/data-management-tutorial.md

```diff
@@ -132,7 +132,7 @@ To export data from Viam:
 1. First, install the [Viam CLI](/manage/cli) and [authenticate](/manage/cli/#authenticate):
 
    ```sh
-   go install go.viam.com/rdk/cli/cmd@latest
```

Comment:
[optional] Suggestion to just remove the code block here since we already link to the Viam CLI page above. Also, it wasn't working until I added ~/go/bin to my PATH, so I had to go to the Viam CLI page anyway to find out I needed to do that